Jenkinsfile

```diff
@@ -465,15 +465,13 @@ pipeline {
                     }
                     post {
                         always {
-                            node('lightweight') {
-                                recordIssues enabledForFailure: true,
-                                             aggregatingResults: true,
-                                             id: "analysis-gcc-centos7",
-                                             tools: [ gcc4(pattern: 'centos7-gcc-build.log'),
-                                                      cppCheck(pattern: 'centos7-gcc-build.log') ],
-                                             filters: [excludeFile('.*\\/_build\\.external\\/.*'),
-                                                       excludeFile('_build\\.external\\/.*')]
-                            }
+                            recordIssues enabledForFailure: true,
+                                         aggregatingResults: true,
+                                         id: "analysis-gcc-centos7",
+                                         tools: [ gcc4(pattern: 'centos7-gcc-build.log'),
+                                                  cppCheck(pattern: 'centos7-gcc-build.log') ],
```

Comment:
I looked into cppcheck and it appears to be a static analyser tool that we don't even use, if that's the case then we should just delete the cppCheck() code here.

Reply:
Perhaps we should...but not sure that needs to be done in this patch

Reply:
ok.  Last time I checked this we weren't using a pattern for gcc4 and cppCheck and that was causing some issues but it looks like it's been changed anyway since then.